### PR TITLE
Update Rust packet forward example script.

### PR DESCRIPTION
**Pull request by manojgop**
…ers.

Signed-off-by: Manoj Gopalakrishnan <manoj.gopalakrishnan@intel.com>

### CODE REVIEW

**Comment by jeffreybshaw** [comment]
> …ers.
> 
> Signed-off-by: Manoj Gopalakrishnan [manoj.gopalakrishnan@intel.com](mailto:manoj.gopalakrishnan@intel.com)

Can you fix the commit message? The line length is really long, which is awkward for maintainers when reviewing the git log. Though we don't enforce this per-se, I prefer to see commit messages like the ones described in [https://www.kernel.org/doc/html/v4.10/process/submitting-patches.html#the-canonical-patch-format](the linux patch submission guide).

File: lang/rs/README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Following are the steps required to build Rust bindings, high level APIs and exa
       If no command line options are passed, the script will be executed with default values used in the script.
    4. Run loopback example:  `cd apis/cne`. Execute [./run_loopback.sh](./run_loopback.sh)
    5. Run echo server example: `cd examples/echo_server`. Execute [./run_echo_server.sh](./run_echo_server.sh)
-   6. Run packet forward example: `cd examples/fwd`. Execute [./run_fwd.h](./run_fwd.h) `<drop|fwd|tx-only|lb>`
+   6. Run packet forward example: `cd examples/fwd`. Execute [./run_fwd.h](./run_fwd.h) `<JSONC config file>` `<drop|fwd|tx-only|lb>`
```
Comment (KeithWiles) [inline lang/rs/README.md:50]: What about '[[drop | rx-only], tx-only, [fwd | forward], [lb | loopback]] we have shorter modes like fwd and lb which needs to be stated?

**Review by jeffreybshaw — COMMENTED** [review]
Other than the commit message, this looks fine to me.

File: lang/rs/examples/fwd/src/main.rs [hunk 2]
```diff
@@ -25,10 +25,10 @@ enum Mode {
     Drop,
     RxOnly,
```
Comment (KeithWiles) [inline lang/rs/examples/fwd/src/main.rs:26]: Strange that we do not have an alias for drop as rx-only or rx-only with alias of drop. Should we define alias here?

**Review by KeithWiles — CHANGES_REQUESTED** [review]
A few nits.

File: lang/rs/examples/fwd/src/main.rs (hunk 2 above)
Comment (manojgop) [reply lang/rs/examples/fwd/src/main.rs:26]: For RxOnly and Drop, Rust clap module automatically enum shows it as drop and rx-only. It doesn't allow to pass "Drop" or "RxOnly" in command line.

File: lang/rs/README.md (hunk 1 above)
Comment (manojgop) [reply lang/rs/README.md:50]: I'll update the ReadME

**Comment by manojgop** [comment]
> > …ers.
> > Signed-off-by: Manoj Gopalakrishnan [manoj.gopalakrishnan@intel.com](mailto:manoj.gopalakrishnan@intel.com)
> 
> Can you fix the commit message? The line length is really long, which is awkward for maintainers when reviewing the git log. Though we don't enforce this per-se, I prefer to see commit messages like the ones described in [the linux patch submission guide](https://www.kernel.org/doc/html/v4.10/process/submitting-patches.html#the-canonical-patch-format).

Updated the commit message

File: lang/rs/examples/fwd/src/main.rs (hunk 2 above)
Comment (KeithWiles) [reply lang/rs/examples/fwd/src/main.rs:26]: I do not understand, but if it is not allowed then I guess it is OK.

Comment (manojgop) [reply lang/rs/examples/fwd/src/main.rs:26]: Rust enum variant should be in camel case.  Rust clap module will change enum variant "RXOnly" to "rx-only" by default.

**Review by KeithWiles — APPROVED** [review]
LGTM

**Pull request merged**